src/screens/TransactionViews/TransactionViewUtils.res

```diff
@@ -0,0 +1,70 @@
+open TransactionViewTypes
+
+let paymentViewsArray: array<viewTypes> = [All, Succeeded, Failed, Dropoffs, Cancelled]
```

Comment:
can you try to use Polymorphic variants?
Because then these mappers won't be required as they are single word only. Try once

Reply:
it will not create too much of a difference, we can pick it as refactor separately, as mapper for api will still be required